build.gradle.kts

```diff
@@ -9,7 +9,10 @@ nexusPublishing {
   packageGroup.set("com.splunk")
 
   repositories {
+    // see https://central.sonatype.org/publish/publish-portal-ossrh-staging-api/#configuration
     sonatype {
+      nexusUrl.set(uri("https://ossrh-staging-api.central.sonatype.com/service/local/"))
+      snapshotRepositoryUrl.set(uri("https://central.sonatype.com/repository/maven-snapshots/"))
```

Comment:
we may need to update https://github.com/signalfx/splunk-otel-java?tab=readme-ov-file#snapshot-builds if snapshots are now in new location

Reply:
agreed. I guess we'll know for sure after the first one is published...because even tho I migrated us, the `com/splunk` snapshots don't yet exist. Maybe/probably just new stuff.